Build failure

```
ocamlbuild -I src -I lib -I parser -I kremlib -use-menhir -use-ocamlfind -classic-display -menhir "menhir --infer --explain" Kremlin.native
ocamlfind ocamlc -c -g -annot -bin-annot -strict-sequence -w @1-2-3@8..12@14..21@23..29-30@31..38-39-40-41@43@57 -package ppx_deriving.std -package ppx_deriving_yojson -package zarith -package pprint -package unix -package menhirLib -package sedlex -package process -package fix -package wasm -package visitors.ppx -package visitors.runtime -I src -I parser -I kremlib -I lib -o src/CFlatToWasm.cmo src/CFlatToWasm.ml
+ ocamlfind ocamlc -c -g -annot -bin-annot -strict-sequence -w @1-2-3@8..12@14..21@23..29-30@31..38-39-40-41@43@57 -package ppx_deriving.std -package ppx_deriving_yojson -package zarith -package pprint -package
unix -package menhirLib -package sedlex -package process -package fix -package wasm -package visitors.ppx -package visitors.runtime -I src -I parser -I kremlib -I lib -o src/CFlatToWasm.cmo src/CFlatToWasm.ml
File "src/CFlatToWasm.ml", line 196, characters 18-32:
196 |   [ dummy_phrase (W.Ast.TeeLocal (mk_var (env.n_args + 2)));
                        ^^^^^^^^^^^^^^
Error: Unbound constructor W.Ast.TeeLocal
Command exited with code 2.
Makefile:22: recipe for target 'minimal' failed
make: *** [minimal] Error 10
```

I am using wasm 1.1, maybe that's the problem?

--- a/README.md
+++ b/README.md
@@ -37,14 +37,14 @@ key component of [Project Everest](https://project-everest.github.io/).
 
 ## Trying out KreMLin
 
-KreMLin requires OCaml (>= 4.05.0, < 4.08.0), OPAM, and a recent version of GNU make.
+KreMLin requires OCaml (>= 4.05.0, < 4.10.0), OPAM, and a recent version of GNU make.
 
 **Regarding GNU make:** On OSX, this may require you to install a recent GNU
 make via homebrew, and invoke `gmake` instead of `make`.
 
 **Regarding OCaml:** Install OPAM via your package manager, then:
 
-`$ opam install ppx_deriving_yojson zarith pprint menhir sedlex process fix wasm visitors ctypes-foreign ctypes`
+`$ opam install ppx_deriving_yojson zarith pprint menhir sedlex process fix wasm.1.0.1 visitors ctypes-foreign ctypes`
 
 Next, make sure you have an up-to-date F\*, and that you ran `make` in the
 `ulib/ml` directory of F\*. The `fstar.exe` executable should be on your PATH

